.github/workflows/build-test-measure.yml

```diff
@@ -68,7 +68,13 @@ jobs:
       - name: Validate package.json
         run: npm run lint:pkg-json
 
-      - name: Detect coding standard violations (ESLint)
+      - name: Detect ESLint coding standard violations
+        if: github.event.pull_request.head.repo.fork == true
```

Comment:
Ah, `npm run lint:js:report` _also_ does the linting, so that is why this is is limited to forks.